bc_obps/reporting/api/reports.py

```diff
@@ -74,6 +79,7 @@ def get_reporting_year(request: HttpRequest) -> Tuple[Literal[200], ReportingYea
     tags=EMISSIONS_REPORT_TAGS,
     description="""Takes `version_id` (primary key of the ReportVersion model) and `facility_id` to return a single matching `facility_report` object.
     Includes the associated activity IDs if found; otherwise, returns an error message if not found or in case of other issues.""",
+    auth=authorize("approved_authorized_roles"),
 )
 @handle_http_errors()
 def get_facility_report_by_version_and_id(
```

Comment:
Just a note for rebasing: these facility_report endpoints have been split into their own file.

Reply:
Rebased and adjusted the locations to the new file.